modules/dedicated-enable-private-cluster.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/dedicated-private-cluster.adoc
+
+[id="dedicated-enable-private-cluster-new"]
+= Enabling Private Cluster on a new cluster
+
+.Prerequisites
+
+* AWS VPC Peering, VPN, DirectConnect, or TransitGateway has been previously configured to allow private access.
+
+.Procedure
+
+. In the OpenShift Cluster Manager, click *Create cluster* and select *{product-title}*.
+. Configure your Cluster details, then select *Advanced* in the Networking section.
+. Determine your CIDR requirements for your network.
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+====
+. Under Cluster Privacy, select *Private*.
```

Comment:
Is Cluster Privacy a window title or a specific UI element, like a drop-down list? If it's a title, the styling's right. If it's not, it most likely needs to be enclosed in asterisks. (Check out the section on graphical user interface elements starting on page 206 in the ISG.)